---
title: Comprehensive Benchmarking and Ensemble Approaches for Metagenomic Classifiers
author: Alexa B. R. McIntyre et al.
date: '2017-09-01'
categories:
  - Papers
tags:
  - Metagenomics
  - Taxonomy classifiers
  - Tool comparisons
  - Pathogen detection
  - Preprints
---

### Alexa B. R. McIntyre, Rachid Ounit, Ebrahim Afshinnekoo, Robert J. Prill, Elizabeth HeÃÅnaff, Noah Alexander, Sam Minot, David Danko, Jonathan Foox, Sofia Ahsanuddin, Scott Tighe, Nur A. Hasan, Poorani Subramanian, Kelly Moffat, Shawn Levy, Stefano Lonardi, Nick Greenfield, Rita R. Colwell, Gail L. Rosen, Christopher E. Mason

**Link:** [http://www.biorxiv.org/content/early/2017/06/28/156919.1](http://www.biorxiv.org/content/early/2017/06/28/156919.1)

**When:** Friday, September 1 at 10:00AM

**Where:** CBCB Room 3118 (Biomolecular Sciences Building Rm 3118)

### Abstract

*Background*: One of the main challenges in metagenomics is the identification of microorganisms in clinical and environmental samples. While an extensive and heterogeneous set of computational tools is available to classify microorganisms using whole genome shotgun sequencing data, comprehensive comparisons of these methods are limited. In this study, we use the largest (n=35) to date set of laboratory-generated and simulated controls across 846 species to evaluate the performance of eleven metagenomics classifiers. We also assess the effects of filtering and combining tools to reduce the number of false positives.

*Results*: Tools were characterized on the basis of their ability to (1) identify taxa at the genus, species, and strain levels, (2) quantify relative abundance measures of taxa, and (3) classify individual reads to the species level. Strikingly, the number of species identified by the eleven tools can differ by over three orders of magnitude on the same datasets. However, various strategies can ameliorate taxonomic misclassification, including abundance filtering, ensemble approaches, and tool intersection. Indeed, leveraging tools with different heuristics is beneficial for improved precision. Nevertheless, these strategies were often insufficient to completely eliminate false positives from environmental samples, which are especially important where they concern medically relevant species and where customized tools may be required.

*Conclusions*: The results of this study provide positive controls, titrated standards, and a guide for selecting tools for metagenomic analyses by comparing ranges of precision and recall. We show that proper experimental design and analysis parameters, including depth of sequencing, choice of classifier or classifiers, database size, and filtering, can reduce false positives, provide greater resolution of species in complex metagenomic samples, and improve the interpretation of results.

**bioRxiv preprint first posted online:** June 28, 2017

### Citation

**Note:** This is in pre-print, so citing this in a published manuscript would be ill-advised until this article has gone through peer review. Still, it can be be cited as follows:

McIntyre ABR, Ounit R, et al. (2017) Comprehensive Benchmarking and Ensemble Approaches for Metagenomic Classifiers. bioRxiv 156919; doi: https://doi.org/10.1101/156919


