<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Co-Enriching Microflora Associated with Culture Based Methods to Detect Salmonella from Tomato Phyllosphere | UMD Microbiome Journal Club</title>
    <link rel="stylesheet" href="/website/css/style.css" />
    <link rel="stylesheet" href="/website/css/fonts.css" />
    
  </head>

  <body>
    <nav>
    <ul class="menu">
      
      <li><a href="/website/">Home</a></li>
      
      <li><a href="/website/meetings/">Meetings</a></li>
      
      <li><a href="/website/resources/">Resources</a></li>
      
      <li><a href="/website/people/">People</a></li>
      
      <li><a href="/website/categories/">Categories</a></li>
      
      <li><a href="/website/tags/">Tags</a></li>
      
    </ul>
    <hr/>
    </nav>

<div class="article-meta">
<h1><span class="title">Co-Enriching Microflora Associated with Culture Based Methods to Detect Salmonella from Tomato Phyllosphere</span></h1>
<h2 class="author">Andrea R. Ottesen et al.</h2>
<h2 class="date">2017/07/28</h2>
</div>

<main>


<h3 id="andrea-r-ottesen-antonio-gonzalez-rebecca-bell-caroline-arce-steven-rideout-marc-allard-peter-evans-errol-strain-steven-musser-rob-knight-eric-brown-james-b-pettengill">Andrea R. Ottesen, Antonio Gonzalez, Rebecca Bell, Caroline Arce, Steven Rideout, Marc Allard, Peter Evans, Errol Strain, Steven Musser, Rob Knight, Eric Brown, James B. Pettengill</h3>

<p><strong>Link:</strong> <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0073079">http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0073079</a></p>

<p><strong>When:</strong> Friday, July 28 at 10:00AM</p>

<p><strong>Where:</strong> MIAEH conference room (SPH 2242CC)</p>

<h3 id="abstract">Abstract</h3>

<p>The ability to detect a specific organism from a complex environment is vitally important to many fields of public health, including food safety. For example, tomatoes have been implicated numerous times as vehicles of foodborne outbreaks due to strains of <em>Salmonella</em> but few studies have ever recovered <em>Salmonella</em> from a tomato phyllosphere environment. Precision of culturing techniques that target agents associated with outbreaks depend on numerous factors. One important factor to better understand is which species co-enrich during enrichment procedures and how microbial dynamics may impede or enhance detection of target pathogens. We used a shotgun sequence approach to describe taxa associated with samples pre-enrichment and throughout the enrichment steps of the Bacteriological Analytical Manualâ€™s (BAM) protocol for detection of <em>Salmonella</em> from environmental tomato samples. Recent work has shown that during efforts to enrich <em>Salmonella</em> (Proteobacteria) from tomato field samples, Firmicute genera are also co-enriched and at least one co-enriching Firmicute genus (<em>Paenibacillus</em> sp.) can inhibit and even kills strains of <em>Salmonella</em>. Here we provide a baseline description of microflora that co-culture during detection efforts and the utility of a bioinformatic approach to detect specific taxa from metagenomic sequence data. We observed that uncultured samples clustered together with distinct taxonomic profiles relative to the three cultured treatments (Universal Pre-enrichment broth (UPB), Tetrathionate (TT), and Rappaport-Vassiliadis (RV)). There was little consistency among samples exposed to the same culturing medias, suggesting significant microbial differences in starting matrices or stochasticity associated with enrichment processes. Interestingly, <em>Paenibacillus</em> sp. (<em>Salmonella</em> inhibitor) was significantly enriched from uncultured to cultured (UPB) samples. Also of interest was the sequence based identification of a number of sequences as <em>Salmonella</em> despite indication by all media, that samples were culture negative for <em>Salmonella</em>. Our results substantiate the nascent utility of metagenomic methods to improve both biological and bioinformatic pathogen detection methods.</p>

<p><strong>Published</strong> September 9, 2013</p>

<h3 id="citation">Citation</h3>

<p>Ottesen AR, Gonzalez A, Bell R, Arce C, Rideout S, et al. (2013) Co-Enriching Microflora Associated with Culture Based Methods to Detect <em>Salmonella</em> from Tomato Phyllosphere. PLoS ONE 8(9): e73079. doi:10.1371/journal.pone.0073079</p>

</main>

  <footer>
  <script src="//yihui.name/js/math-code.js"></script>
<script async src="//cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML"></script>

<script async src="//yihui.name/js/center-img.js"></script>

  
  <hr/>
  &copy; Dan Nasko 2017 | <a href="https://github.com/dnasko">Github</a> | <a href="https://twitter.com/dnasko">Twitter</a>
  
  </footer>
  </body>
</html>

